lib/fastlane/plugin/react_native_release/actions/react_native_release_action.rb

```diff
@@ -129,24 +127,25 @@ def self.create_fastlane_session()
 
         UI.message "Generating a new FASTLANE_SESSION."
 
-        file = Tempfile.new('')
-
         fastlane_session_git_url =  ENV["FASTLANE_ENV_GIT_URL"]
         fastlane_session_username = ENV["FASTLANE_ENV_USERNAME"]
 
+        spaceship_cookie_path = "#{File.expand_path('~')}/.fastlane/spaceship/#{fastlane_session_username}/cookie"
```

Comment:
is there by chance a fastlane env that maps to `~/.fastlane`? if so that might be better to depend on.

Reply:
Not to `~/.fastlane`.
I thought about using `ENV["HOME"]` but I'm not sure exactly if I can/should use that.